pyaedt/generic/settings.py

```diff
@@ -0,0 +1,528 @@
+        If ``True`` the property or method will return a pandas object in CPython environment.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_pandas_output
+
+    @enable_pandas_output.setter
+    def enable_pandas_output(self, val):
+        self._enable_pandas_output = val
+
+    @property
+    def enable_debug_methods_argument_logger(self):
+        """
+        Set/Get a flag to plot methods argument in debug logger.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_debug_methods_argument_logger
+
+    @enable_debug_methods_argument_logger.setter
+    def enable_debug_methods_argument_logger(self, val):
+        self._enable_debug_methods_argument_logger = val
+
+    @property
+    def force_error_on_missing_project(self):
+        """Set/Get a flag to check project path.
+        If ``True`` when passing a project path, the project has to exist otherwise it will raise an error.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._force_error_on_missing_project
+
+    @force_error_on_missing_project.setter
+    def force_error_on_missing_project(self, val):
+        self._force_error_on_missing_project = val
+
+    @property
+    def disable_bounding_box_sat(self):
+        """Set/Get Bounding Box Sat enablement.
+
+        Returns
+        -------
+        bool
+        """
+        return self._disable_bounding_box_sat
+
+    @disable_bounding_box_sat.setter
+    def disable_bounding_box_sat(self, val):
+        self._disable_bounding_box_sat = val
+
+    @property
+    def use_grpc_api(self):
+        """Set/Get GRPC API usage or Legacy COM Object. This is the user setting.
+
+        Returns
+        -------
+        bool
+        """
+        return self._use_grpc_api
+
+    @use_grpc_api.setter
+    def use_grpc_api(self, val):
+        """Set/Get GRPC API usage or Legacy COM Object. This is the user setting."""
+        self._use_grpc_api = val
+
+    @property
+    def logger(self):
+        """Get the active logger."""
+        try:
+            return logging.getLogger("Global")
+        except:  # pragma: no cover
+            return logging.getLogger(__name__)
+
+    # @property
+    # def non_graphical(self):
+    #     """Get the value for the non-graphical flag."""
+    #     return self._non_graphical
+    #
+    # @non_graphical.setter
+    # def non_graphical(self, val):
+    #     self._non_graphical = val
+
+    @property
+    def enable_error_handler(self):
+        """Return the content for the environment variable."""
+        return self._enable_error_handler
+
+    @enable_error_handler.setter
+    def enable_error_handler(self, val):
+        self._enable_error_handler = val
+
+    @property
+    def enable_desktop_logs(self):
+        """Get the content for the environment variable."""
```

Comment:
Is this description wrong due to a copy and paste error. It seems like it should be "Flag indicating whether desktop logs are enabled."